markets/perps-market/contracts/modules/PerpsAccountModule.sol

```diff
@@ -94,14 +96,21 @@ contract PerpsAccountModule is IPerpsAccountModule {
 
     // 1. call depositMarketUsd and deposit amount directly to core system
     // 2. look up account and reduce debt by amount
-    // 3b. quoteUnwrap() -> inchQuote -> returnAmount
     function payDebt(uint128 accountId, uint256 amount) external override {
+        FeatureFlag.ensureAccessToFeature(Flags.PERPS_SYSTEM);
```

Comment:
Worth noting/considering that the BFP market validates that the account has permission to modify the account's collateral: https://github.com/Synthetixio/synthetix-v3/blob/8aff01938913983b97faa5ce082c15b86db32e0d/markets/bfp-market/contracts/modules/MarginModule.sol#L498

Reply:
Additionally, BFP will revert on 0 amounts, don't see any bad outcomes here, but avoids unexpected behaviors/event emissions.

Reply:
I suppose even if you reverted on 0 amounts here, the debtPaid could still be capped to 0 too. So if added, perhaps the validation should be based on the `debtPaid` result.